day04/generateArray.js

```diff
@@ -0,0 +1,2 @@
+exports.generateArray = (start, end) =>
+  Array.from({ length: end + 1 - start }, (x, i) => i + start);
```

Comment:
I think this `end +1 - start` is just equal to `end`!

Reply:
is there a ramda function that will generate a range like this?

Reply:
There is, thanks for the tip. R.range does exactly this.